### PR TITLE
Add tag deletion from tags list

### LINKED ISSUE #2952: Remove tag from tag/list
Hi,
It would be great to be able to remove tags from the tag/list view. I've got 87 tags generated automatically and I don't want to go to each article to remove it. Let me know if I missed something.

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| CHANGELOG.md  | yes/no
| License       | MIT

Fixed #2952

Need help for design (@Simounet maybe?)

### CODE REVIEW

**Review by Kdecherf — CHANGES_REQUESTED** [review]
Could you add a screenshot and a prompt for asking the user to confirm the deletion?

**Comment by Simounet** [comment]
Here is the screen. I used a JS `confirm()` but this is not a good practice so if I'm motivated enough, I'll do another PR to replace the `confirm()`'s by a better handler like [a11ydialog](https://a11y-dialog.netlify.app/) but I don
tag-delete-action.

**Comment by nicosomb** [comment]
Thank you @Simounet

**Comment by yguedidi** [comment]
@nicosomb according to the milestone set, shouldn't this PR target the `2.6.0` branch instead of `master`? or change the milestone of course! :)

File: tests/Wallabag/CoreBundle/Controller/TagControllerTest.php [hunk 1]
```diff
@@ -141,6 +141,53 @@ public function testRemoveTagFromEntry()
...
+        $this->getEntityManager()->persist($entry);
+
+        $entry2 = new Entry($this->getLoggedInUser());
+        $entry2->setUrl('http://0.0.0.0/bar');
+        $entry2->addTag($tag);
+        $this->getEntityManager()->persist($entry2);
+        $this->getEntityManager()->flush();
+        $this->getEntityManager()->clear();
+
+        $client->request('GET', '/tag/delete/' . $tag->getSlug());
```
Comment (yguedidi) [inline tests/Wallabag/CoreBundle/Controller/TagControllerTest.php:164]: @nicosomb wouldn't it be better to use a POST request for an action that alters the database?

Comment (nicosomb) [reply tests/Wallabag/CoreBundle/Controller/TagControllerTest.php:164]: You're right ... for the API side. 

But here, it's not the API part. Our tests are web test cases. We just simulate a click on the link.

Comment (yguedidi) [reply tests/Wallabag/CoreBundle/Controller/TagControllerTest.php:164]: I think even in interfaces it should be a POST action :)
using a form and a submit button.
IMO it's a good practice the have all actions that alter the database to be POST actions

Comment (j0k3r) [reply tests/Wallabag/CoreBundle/Controller/TagControllerTest.php:164]: Why don't you use the client and then the crawler to retrieve the button and simulate a click on it?

Comment (nicosomb) [reply tests/Wallabag/CoreBundle/Controller/TagControllerTest.php]: @j0k3r done

**Review by Kdecherf — APPROVED** [review]

**Pull request merged**